### PR TITLE
LPS-166740 asset entry is linked to the classPK that changes if the kbArticle is approved or not

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
*Step to reproduce:*

1. Enable KB FF
2. Create a display page template for knowledge base articles
3. Add a KB article 
4. Select this added Display page template for this KB article and publish it
5. Delete this KB article
6. Go to Page Templates > Display Page Templates to delete the KB display page template

*Expected Results:*
Able to delete it.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java [hunk 1]
```diff
@@ -138,8 +138,7 @@ else if (cmd.equals(Constants.UPDATE)) {
 		}
 
 		_assetDisplayPageEntryFormProcessor.process(
-			KBArticle.class.getName(), kbArticle.getKbArticleId(),
-			actionRequest);
+			KBArticle.class.getName(), kbArticle.getClassPK(), actionRequest);
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java:141]: This cannot be right. The `KBArticleLayoutDisplayPageObjectProvider` and `KBArticleLayoutDisplayPageProvider` use the `kbArticleId`. Maybe there's an inconsistency somewhere else?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/portlet/action/UpdateKBArticleMVCActionCommand.java:141]: We'll probably need to change everything to the class pk :-(

**Comment by AliciaGarciaGarcia** [comment]
@adolfopa I made another approach of this on https://github.com/liferay-lima/liferay-portal/pull/3666

**Comment by adolfopa** [comment]
Closing in favor of https://github.com/liferay-lima/liferay-portal/pull/3858

**Pull request closed without merging**